cnd/src/swap_protocols/facade.rs

```diff
@@ -21,18 +22,22 @@ pub struct HbitHerc20SwapParams {
     pub peer: DialInformation,
     #[digest(ignore)]
     pub bitcoin_identity: identity::Bitcoin,
-    #[digest(prefix = "2001")]
+    #[digest(prefix = "e6ca")]
```

Comment:
Would the macro work with constants @D4nte? Could aid documentation a lot to export this as constants in a single file so we can see, where they are reused.

Reply:
We only support a string literal right now but shouldn't be too hard to change [this](https://github.com/comit-network/comit-rs/blob/80b019cde297d3d2ce88f3253b92678ceeeadf20/digest-macro-derive/src/lib.rs#L168) `if let` with a `match` to support more types.